virttest/utils_libvirt/libvirt_ceph_utils.py

```diff
@@ -0,0 +1,154 @@
+"""
+High-level libvirt ceph utility functions.
+
+This module is meant to reduce code size by performing common ceph setup procedures.
+:copyright: 2021 Red Hat Inc.
+"""
+
+import os
+import logging
+
+from avocado.utils import process
+from avocado.core import exceptions
+
+from virttest import data_dir
+from virttest import virsh
+from virttest import utils_package
+from virttest import ceph
+from virttest import utils_secret
+
+from virttest.utils_test import libvirt
+
+from virttest.libvirt_xml import vm_xml
+from virttest.libvirt_xml.devices.disk import Disk
+
+
+def create_or_cleanup_ceph_backend_vm_disk(vm, params, is_setup=True):
+    """
+    Setup vm ceph disk with given parameters
+
+    :param vm: the vm object
+    :param params: dict, dict include setup vm disk xml configurations
+    :param is_setup: one parameter indicate whether setup or clean up
+    """
+    vmxml = vm_xml.VMXML.new_from_dumpxml(vm.name)
+    logging.debug("original xml is: %s", vmxml.xmltreefile)
```

Comment:
Why we're logging vmxml.xmltreefile instead of vmxml ?

Reply:
I see both are used somewhere ,and both can achieve full xml ouput

Reply:
I know that, but we tend to keep our code short and clear. So I'd suggest we use vmxml instead of vmxml.xmltreefile.

Reply:
> I know that, but we tend to keep our code short and clear. So I'd suggest we use vmxml instead of vmxml.xmltreefile.

isn't  vmxml.xmltreefile more meaningful than vmxml when talking about dumpxing files.

Reply:
I don't think so. But keep it the way it is if you'd like. 
I just prefer shorter code which is more elegant.

Reply:
> @chloerh @chunfuwen I agree it should be non-blocking but I would prefer not using xmltreefile. I believe the intention of VMXML is to use vmxml directly not the xmltreefile instance, I deduce this because the VMXML implements the `__str__` method which is used for string representation of an object:
> https://github.com/avocado-framework/avocado-vt/blob/master/virttest/libvirt_xml/base.py#L54
> https://docs.python.org/3/library/stdtypes.html#textseq
> And, as pointed out by Haijiao, it's also shorter.

@smitterl , although I has different opinion with you, I will update this.